vhdbuilder/packer/components.json

```diff
@@ -466,7 +467,8 @@
       "versions": [
         "1.2.7",
         "1.4.12",
```

Comment:
her keep 12 but remove defaultValue: "https://acs-mirror.azureedge.net/azure-cni/v1.4.12/binaries/azure-vnet-cni-swift-linux-amd64-v1.4.12.tgz"

Reply:
@paulgmiller Charlie shared "arm64 and amd64 is using swift cni 1.4.12" so it seems like that it should be kept.